examples/api-tests/src/typescript.spec.js

```diff
@@ -629,7 +629,7 @@ SPAN {
         const editor = await openEditor(demoFileUri);
         const currentChar = () => editor.getControl().getModel().getLineContent(lineNumber).charAt(column - 1);
 
-        // const demoVariable = demoInstance.stringField; --> const demoVariable = demoInstance.stringFiel;
+        // const demoVariable = demoInstance.stringField; --> const demoVariable = demoInstance.stringField;
```

Comment:
This is deliberate: the test in fact removes the 'd' from the end of `stringField` to trigger an autofix suggestion.